inspirehep/utils/helpers.py

```diff
@@ -78,3 +78,12 @@ def force_force_list(data):
     elif isinstance(data, (tuple, set)):
         return list(data)
     return data
+
+def is_deleted(data):
+    """Check if data is deleted.
+    """
+
+    if data.lower() == 'deleted':
```

Comment:
You have to check that `data` is actually different than `None`. (this is what is making the whole PR to crash BTW :dancer: )
So you can do e.g. `if data and data.lower() == 'deleted'.